packages/request-utils/src/index.ts

```diff
@@ -52,7 +52,7 @@ type Store = {
 // host and namespace which are provided by the final consuming
 // class to the prototype which can result in overwrite errors
 
-interface BuildURLConfig {
+export interface BuildURLConfig {
```

Comment:
@runspired I have a question, do you think it would be useful to allow users to overwrite function that does `resourcePath` conversion right here as part of BuildURLConfig? 

Something like

```ts 
BuildURLConfig({
  host: "https://myhost.com",
  namespace: "api/v1",
  resourcePath: (resourceName) => {
    return camelize(pluralize(resourceName));
  }
});
``` 

or is it too crazy and I should go to sleep ?

Reply:
go to sleep :P

This is mostly already configurable in this way though maybe we could make it even more configurable

Reply:
Yeah I know you can configure it using builder options, but would be nice to have it in single place like it used to be in adapter. It would cover 90% cases, and you can still change it on builder level